bool hasFace
bool isMultiface
sensor_msgs/Image FaceImage


